src/coffee/bootstrap-tour.spec.coffee

```diff
@@ -568,7 +569,26 @@ describe 'Bootstrap Tour', ->
       path: 'test.html'
 
     # Tour adds basePath to step path
-    expect(@tour._isRedirect(@tour._options.basePath + @tour.getStep(0).path, 'test/test.html')).toBe false
+    expect(
+      @tour._isRedirect(
+        @tour.getStep(0).host, @tour._options.basePath + @tour.getStep(0).path, 'test/test.html'
+      )
+    )
+    .toBe false
+
+  it 'should redirect to the steps if host is different', ->
+    @tour = new Tour
+    @tour.addStep
+      element: $('<div></div>').appendTo('body')
+      path: 'test.html'
+      host: 'http://sub.exemple.com'
+
+    expect(
+      @tour._isRedirect(
+        @tour.getStep(0).host, @tour._options.basePath + @tour.getStep(0).path, 'test.html'
+      )
+    )
+    .toBe true
```

Comment:
I think it's already done with this test https://github.com/tekxl/bootstrap-tour/blob/f2fdd998eee4543afa79e2d3d1cb2d4cdb662c33/src/coffee/bootstrap-tour.spec.coffee#L399